gooddata-dbt/gooddata_dbt/dbt/tables.py

```diff
@@ -405,9 +444,9 @@ def make_date_datasets(self, table: DbtModelTable, existing_date_datasets: List[
                 date_datasets.append(
                     {
                         "id": column.ldm_id,
-                        "title": self.get_ldm_title(column),
+                        "title": column.gooddata_ldm_title,
                         "description": column.description,
-                        "tags": [table.gooddata_ldm_title] + column.tags,
+                        "tags": ["Date dimensions"],
```

Comment:
Is this tag by purpose?

Reply:
Yes, unfortunately.
I added a description (from the commit message) here.